admin/class-fastcgi-purger.php

```diff
@@ -168,4 +168,19 @@ public function purge_all() {
 
 	}
 
+	/**
+	 * Constructs the base url to call when purging using the `get_request` method.
+	 *
+	 * @return string
+	 */
+	private function purge_base_url() {
+		$parse = parse_url( site_url() );
```

Comment:
Please use WP function for parse URL. `wp_parse_url()`. Also need to use `home_url()` instead of `site_url()` because home & site URL can be different.

`$parse = wp_parse_url( home_url() );`

Reply:
Should I also change this line to use `site_url`?

https://github.com/stayallive/nginx-helper/blob/839100c8acc01ef15c9250f3693e7692985e1f2a/admin/class-fastcgi-purger.php#L94

I think (although it was 2 years ago) I got it form there and that's why I used `site_url` instead of `home_url`.

Reply:
Oh yes, please. Thanks.